src/app/utils/route-utils.ts

```diff
@@ -37,6 +37,14 @@ export type SpecifiedPerEnabledLayer<T = any, ExcludeLayers = never> = {
 
 export type SpecifiedPerEnabledRuntime<T = any> = SpecifiedPerEnabledLayer<T, typeof Layer.consensus>
 
+export const hiddenLayers: Layer[] = [Layer.pontusxdev]
```

Comment:
what's a plan for pontusxdev? It's going to be accessible via direct url only?

Reply:
Yes, in our release, it's only available via direct URL. (On the pontus-x branch, it's not hidden, so it's available in the layer selector.)